gitnamehere/point-of-sale-group-project
# Orders Screen Part 3

This PR implements order filtering by paid, unpaid, void, and all

Testing:
* `npm run setup-demo:dev` (if you haven't updated your db in a week)
* `npm run start:dev`
* create some orders in the POS
* log into the postgres database with `psql --username postgres` (or whatever your postgres username is)
* \c into atspos
* set some orders to be voided `UPDATE orders SET is_void = true WHERE id = (some order id);`
* go to http://localhost:3000/pos/orders/
* filter by void or all
* verify the order has a void style and VOIDED is displayed
* pay for an unpaid order (white square)
* filter by paid/all, and verify the order has a paid style
* verify a paid order doesn't show up in unpaid orders, and vice versa
* verify a voided order doesn't show up in unpaid orders, and vice versa
